src/components/results/SingleResult.vue

```diff
@@ -0,0 +1,229 @@
+<script setup lang="ts">
+import formattingDate from "../../utils/dateFormatting";
+import ListItem from "./ListItem.vue";
+import { computed } from "vue";
+
+let result = {
+  questionAnswer: [
+    {
+      question: "question1",
+      questionScore: 1,
+      answerPoints: 1,
+      category: "category 1",
+      answer: "answer 1",
+    },
+    {
+      question: "question2",
+      questionScore: 5,
+      answerPoints: 3,
+      category: "category 2",
+      answer: "answer 2",
+    },
+    {
+      question: "question3",
+      questionScore: 1,
+      answerPoints: 0,
+      category: "category 3",
+      answer: "answer 3",
+    },
+    {
+      question: "question4",
+      questionScore: 5,
+      answerPoints: 4,
+      category: "category 4",
+      answer: "answer 4",
+    },
+    {
+      question: "question5",
+      questionScore: 5,
+      answerPoints: 0,
+      category: "category 5",
+      answer: "answer 5",
+    },
+    {
+      question: "question6",
+      questionScore: 1,
+      answerPoints: 1,
+      category: "category 6",
+      answer: "answer 6",
+    },
+    {
+      question: "question7",
+      questionScore: 5,
+      answerPoints: 2,
+      category: "category 7",
+      answer: "answer 7",
+    },
+  ],
+  startedAt: 1668092016,
+  endedAt: 1668092016,
+  title: "title 1",
+  id: "1",
+  candidateId: "1",
+  parent: {
+    position: "position 1",
+    username: "username 1",
+    linkedinUrl: "linkedinUrl 1",
+    feedback: "feedback 1",
+    avatarUrl:
+      "https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/602.jpg",
+    id: "1",
+  },
+};
+
+const correctAnswers = computed(() => {
+  return result.questionAnswer.filter(
+    (obj) => obj.questionScore === obj.answerPoints
+  ).length;
+});
+
+const partiallyAnswers = computed(() => {
+  return result.questionAnswer.filter(
+    (obj) => obj.answerPoints != 0 && obj.answerPoints < obj.questionScore
+  ).length;
+});
+
+const wrongAnswers = computed(() => {
+  return result.questionAnswer.filter((obj) => obj.answerPoints === 0).length;
+});
+
+const totalQuestions = computed(() => {
```

Comment:
it's fine, but as an option all these short computeds might be a bit shorted like
```
const totalQuestions = computed(() => result.questionAnswer.length)
```